docs/standalone-transaction.md

```diff
@@ -0,0 +1,69 @@
+In this guide, we will generate some wealth for the `Fetch.ai testnet` and create a standalone transaction. After the completion of the transaction,
+we get the transaction digest. With this we can search the transaction it in the <a href='https://explore-testnet.fetch.ai'>block explorer</a>
+
+## Create the wallets
+
+We will need to create two different addresses for this demo. To do this we instantiate two different wallets 
+
+```python
+    wallet_1 = Wallet({FETCHAI: FETCHAI_PRIVATE_KEY_FILE})
```

Comment:
use dynamically created key files, also add the import statements at the beginning for all required modules and constants

Reply:
I am not sure what you mean here.

Reply:
I think I found it.